templates/ctl-scaffold/packages.dhall

```diff
@@ -0,0 +1,368 @@
+          , "safely"
+          , "spec"
+          , "strings"
+          , "these"
+          , "transformers"
+          , "typelevel"
+          , "tuples"
+          , "unicode"
+          , "unordered-collections"
+          , "unsafe-coerce"
+          ]
+        , repo = "https://github.com/juspay/medea-ps.git"
+        , version = "8b215851959aa8bbf33e6708df6bd683c89d1a5a"
+        }
+      , purescript-toppokki =
+        { dependencies =
+          [ "prelude"
+          , "record"
+          , "functions"
+          , "node-http"
+          , "aff-promise"
+          , "node-buffer"
+          , "node-fs-aff"
+          ]
+        , repo = "https://github.com/firefrorefiddle/purescript-toppokki"
+        , version = "6983e07bf0aa55ab779bcef12df3df339a2b5bd9"
+        }
+      , cardano-transaction-lib =
+        { dependencies =
+          [ "aeson"
+          , "aeson-helpers"
+          , "aff"
+          , "aff-promise"
+          , "aff-retry"
+          , "affjax"
+          , "arraybuffer-types"
+          , "arrays"
+          , "bifunctors"
+          , "bigints"
+          , "checked-exceptions"
+          , "console"
+          , "const"
+          , "contravariant"
+          , "control"
+          , "datetime"
+          , "debug"
+          , "effect"
+          , "either"
+          , "encoding"
+          , "enums"
+          , "exceptions"
+          , "foldable-traversable"
+          , "foreign"
+          , "foreign-object"
+          , "http-methods"
+          , "identity"
+          , "integers"
+          , "js-date"
+          , "lattice"
+          , "lists"
+          , "maybe"
+          , "medea"
+          , "media-types"
+          , "monad-logger"
+          , "mote"
+          , "newtype"
+          , "node-buffer"
+          , "node-child-process"
+          , "node-fs"
+          , "node-fs-aff"
+          , "node-path"
+          , "node-process"
+          , "node-streams"
+          , "nonempty"
+          , "optparse"
+          , "now"
+          , "numbers"
+          , "ordered-collections"
+          , "orders"
+          , "parallel"
+          , "partial"
+          , "posix-types"
+          , "prelude"
+          , "profunctor"
+          , "profunctor-lenses"
+          , "purescript-toppokki"
+          , "quickcheck"
+          , "quickcheck-combinators"
+          , "quickcheck-laws"
+          , "rationals"
+          , "record"
+          , "refs"
+          , "spec"
+          , "spec-quickcheck"
+          , "strings"
+          , "tailrec"
+          , "text-encoding"
+          , "these"
+          , "transformers"
+          , "tuples"
+          , "typelevel"
+          , "typelevel-prelude"
+          , "uint"
+          , "undefined"
+          , "unfoldable"
+          , "untagged-union"
+          , "variant"
+          ]
+        , repo = "https://github.com/Plutonomicon/cardano-transaction-lib.git"
+        , version = "ec38c06c77c1b75a7e7f1dcb215db7816e44b339"
```

Comment:
What's the policy on when we should update this revision?

Reply:
We can also discuss that in #839. I think we should decide that it needs to be within a certain number of commits on `develop`

Reply:
Then I can add something to the `checks` to enforce this